cumulusci/tasks/salesforce/users/permsets.py

```diff
@@ -19,18 +23,42 @@ def _init_options(self, kwargs):
 
         self.options["api_names"] = process_list_arg(self.options["api_names"])
 
+    @property
+    def permission_name(self):
+        return "PermissionSet"
+
+    @property
+    def permission_name_field(self):
+        return "Name"
+
+    @property
+    def permission_label(self):
+        return "Permission Set"
+
+    @property
+    def assignment_name(self):
+        return "PermissionSetAssignment"
+
+    @property
+    def assignment_lookup(self):
+        return "PermissionSetId"
+
+    @property
+    def assignment_child_relationship(self):
+        return "PermissionSetAssignments"
```

Comment:
There is no reason for these to be properties. You can just set a class attribute, e.g. 

    assignment_child_relationship = "PermissionSetAssignments"

Reply:
Done.  Change is in upcoming commit.

Reply:
Change is commited.